helpers/lib/cdnify.js

```diff
@@ -30,6 +30,16 @@ module.exports = globals => {
             return path;
         }
 
+        if (['.', '/'].includes(path[0])) {
```

Comment:
In general, modifying a string that you are iterating through is a bad practice. A much cleaner implementation would be:

```
path = path.replace(/^[\.\/]+/, '');
```

This is similar to your previous solution. My comment at that time was to make the characters you are selecting for more explicit -- i.e. to use `[\.\/]` instead of `[^a-zA-Z0-9]`.